### PR TITLE
PN-5276

### PR DESCRIPTION
**Pull request by nathancomstock**
Replaced with https://github.com/wf2/yui3/pull/7
## PN-5276

support shortened combo urls via alias

### CODE REVIEW

File: src/loader/scripts/wf2-create-alias.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  wf2Alias;
+
+wf2Alias = fs.readFileSync(path.join(__dirname, '..', '..') + '/wf2-alias.json');
+
+var alias = JSON.parse(wf2Alias || '[]');
+var arr = [];
+
+//remove entries without a key or value
+alias.forEach(function(item, index) {
+  if(alias[index].key && alias[index].module) {
```
Comment (gerardkcohen) [inline src/loader/scripts/wf2-create-alias.js]: Doesn't item === alias[index]?

Comment (gerardkcohen) [inline src/loader/scripts/wf2-create-alias.js]: On second thought, might be cleaner to use a filter.

``` javascript
var arr = alias.filter(function (item) {
    return item.key && item.module;
});
```

File: src/loader/scripts/wf2-create-alias.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  if(thisLen > thatLen) {
+    return -1;
+  }
+  if(thisLen === thatLen) {
+    return 0;
+  }
+  return 1;
+}
+arr.sort(lengthComparitor);
```
Comment (gerardkcohen) [inline src/loader/scripts/wf2-create-alias.js:32]: Would this slightly shorter version work?

``` javascript
arr.sort(function (a,b) {
    return (a.length < b.length) ? 1 : 0;
 });
```

File: src/loader/scripts/wf2-create-alias.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+// populate key and value arrays
+arr.forEach(function(item, index) {
```
Comment (gerardkcohen) [inline src/loader/scripts/wf2-create-alias.js]: Remove index, as you might get a lint error since it is not used.

Comment (nathancomstock) [inline src/loader/scripts/wf2-create-alias.js]: removed. thanks

File: src/loader/scripts/wf2-create-alias.js (hunk 2 above)
Comment (nathancomstock) [inline src/loader/scripts/wf2-create-alias.js:32]: as discussed , this is will not work.

**Pull request closed without merging**